featuretools/primitives/standard/transform_primitive.py

```diff
@@ -387,7 +390,7 @@ class TimeSince(TransformPrimitive):
         [-1000, -1000000000, -120000000000]
     """
     name = 'time_since'
-    input_types = [Datetime]
+    input_types = [[DatetimeTimeIndex], [Datetime]]
```

Comment:
this change looks unrelated

Reply:
Yes, unrelated. I had checked out this file from master, not main. Reverted.